src/NuGet.Indexing/DocumentCreator.cs

```diff
@@ -264,14 +264,32 @@ static void AddSupportedFrameworks(Document document, IDictionary<string, string
                 }
             }
         }
+        static void AddRequiredField(Document document, string destName, IDictionary<string, string> package, List<string> errors, string sourceName, Field.Index fieldIndex, float boost = 1.0f)
```

Comment:
private static void (and newline above this method), also try to break the long lines, so the whole line is readable when reviewing the PR